assets/src/edit-story/components/canvas/canvasProvider.js

```diff
@@ -121,21 +121,6 @@ function CanvasProvider({ children }) {
     ]
   );
 
-  const selectIntersection = useCallback(
-    ({ x: lx, y: ly, width: lw, height: lh }) => {
-      const newSelectedElementIds = currentPage.elements
-        .filter(({ isBackground }) => !isBackground)
-        .filter(({ x, y, width, height }) => {
-          return (
-            x <= lx + lw && lx <= x + width && y <= ly + lh && ly <= y + height
-          );
-        })
-        .map(({ id }) => id);
-      setSelectedElementsById({ elementIds: newSelectedElementIds });
-    },
-    [currentPage, setSelectedElementsById]
-  );
```

Comment:
Hm, I still think it would be nicer to have the function defined here - or in a reusable hook. This function signature still has all the information needed to do the calculation here, no?

Reply:
@barklund PTAL again.